app/models/file_set.rb

```diff
@@ -1,4 +1,21 @@
 # Generated by hyrax:models:install
 class FileSet < ActiveFedora::Base
   include ::Hyrax::FileSetBehavior
+
+  def language
```

Comment:
rename this to avoid colliding with the `language` property already present on a FileSet